community/doc_guidelines/rest_apis.md

```diff
@@ -0,0 +1,75 @@
+# Documentation Guidelines for REST APIs
+
+<!--
+  Copyright 2018-2020 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+
+Use these guidelines for Splinter REST API (OpenAPI) documentation.
+
+* Follow Splinter's [general documentation guidelines](general.md) and
+  [capitalization guidelines](capitalization.md).
+
+* Include both a summary and a description for each method (GET, POST, etc.).
+
+    - For the method summary, use a very short phrase. Don't add final
+      punctuation.
+
+    - For the method description, use a full sentence that ends with a period.
+      Feel free to use multiple sentences or a few paragraphs, if necessary.
+
+    - For example:
+
+    ```
+    summary: Lists all doohickeys
+    description: Fetches a list of doohickeys for the thingamajig.
+    ```
+
+    <br>
+
+* Use consistent wording for the HTTP verbs. For example:
+
+  <table>
```

Comment:
This section does not format very nicely without adding in css rules. See gameroom_db_table for examples

Reply:
<img width="535" alt="Screen Shot 2020-06-30 at 8 38 14 AM" src="https://user-images.githubusercontent.com/18215486/86132827-11dc9a00-baad-11ea-9948-aa8df4136987.png">

Reply:
Changed to a nested bullet list